mloop_config.py

```diff
@@ -1,110 +1,135 @@
 import os
 import json
 import configparser
+import tomli
 import logging
+from collections import namedtuple
 
 
-def get(config_path=None):
+MloopParam = namedtuple("MloopParam", ["name", "min", "max", "start"])
+RunmanagerGlobal = namedtuple("RunmanagerGlobal", ["name", "expr", "args"])
+
+def is_global_active(config, group, category)

```

Comment:
Have you tried out this code? I would've thought that a missing colon like this would cause an "unexpected indent" error or something like that at import time. I had the impression that you've been using this, so maybe there's a commit which fixes this but hasn't been pushed to github yet?

```suggestion
def is_global_active(config, group, category):
```